src/language-css/parser-postcss.js

```diff
@@ -122,22 +122,18 @@ function parseValueNode(valueNode, options) {
       parenGroupStack.pop();
       parenGroup = getLast(parenGroupStack);
     } else if (node.type === "comma") {
-      if (commaGroup.groups.length > 1) {
-        for (const group of commaGroup.groups) {
-          // if css interpolation
-          if (
-            group.value &&
-            typeof group.value === "string" &&
-            group.value.includes("#{")
-          ) {
-            commaGroup.groups = [
-              stringifyNode({
-                groups: commaGroup.groups,
-              }).trim(),
-            ];
-            break;
-          }
-        }
+      // if SCSS interpolation
+      if (
+        commaGroup.groups.some(
+          (group) =>
+            typeof group.value === "string" && group.value.includes("#{")
+        )
+      ) {
+        commaGroup.groups = [
+          stringifyNode({
+            groups: commaGroup.groups,
+          }).trim(),
+        ];
```

Comment:
This change is not related, but the logic can be simplified to use `Array#some()`.